Generators/Generators.gradle

```diff
@@ -314,6 +328,10 @@ tasks.register 'generateAll', {
     t.description = 'Runs all code generators'
 }
 
+tasks.register 'generatePythonPlot2', {
+    Task t -> t.dependsOn ':Generators:generatePythonFigureWrapper2'
+}
```

Comment:
The syntax is weird, but all this does is create a task that basically calls another task. Since we're actually in :Generators already, it may not be necessary to have this at all - it basically is just an alias for generatePythonFigureWrapper2